### PR TITLE
Double 1 grows 1kgroups

### PR DESCRIPTION
**Pull request by bkietz**
Adds a file containing a single column of `0.0`. There are `2^30` total rows divided between `2^10` row groups. This is useful for testing incremental scanning and materialization; naive load (`read_table`) of this file yields a memory footprint of 25GB on my machine

### CODE REVIEW

**Comment by wesm** [comment]
Having "difficult" files in the testing repo (as long as they aren't too big) probably isn't a bad idea. Do we want to start a README that explains what each file is?

**Comment by bkietz** [comment]
I'll add one

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
...
\ No newline at end of file
+# Testing Data and Utilities for Apache Parquet
+
+## Difficult datasets
+
+Some datasets are included which are tricky to handle:
+
+| dataset name | comments |
+|---|---|
+| `double_1Grows_1kgroups.parquet` | A single column of repeated `0.0`. There are `2^30` total rows divided between `2^10` row groups. This is useful for testing incremental scanning and materialization; naive load (using `pyarrow.parquet.read_table`) of this file yields a memory footprint of 25GB on my machine |
```
Comment (wesm) [inline README.md]: nit: remove "on my machine"

**Comment by bkietz** [comment]
This file is no longer necessary because we generate it in our test case. I'll close this PR unless you'd like to keep the change to README

**Comment by wesm** [comment]
No problem, I'll go ahead and close

**Pull request closed without merging**